goto/commands/use.py

```diff
@@ -0,0 +1,42 @@
+import codecs
+import os
+from ..gotomagic.text import GotoError, GotoWarning
+
+GOTOPATH = os.environ.get('GOTOPATH', os.path.expanduser('~/.goto'))
+
+def use(magic, command, args):
+
+    if len(args) == 0:
+        return None, GotoWarning('missing_project_name', command=command)
+
+    project_name = args[0]
+    project_file = GOTOPATH+'/projects/'+project_name
```

Comment:
This will be refactored soon to a new structure.

`GOTOPATH/projects/<project_name>/<scope>/<project_name>.json`

The old single jfile would correspond to this one
`GOTOPATH/projects/<project_name>/private/<project_name>.json`

These would also be added:
`GOTOPATH/projects/<project_name>/shared/<project_name>.json`
`GOTOPATH/projects/<project_name>/public/<project_name>.json`

What all this means is that we would soon be in a dire need of building a service to create these paths. 
Currently I am using the GotoMagic as that service. (plus som `gotomagic.utils` functions. )
Ref: #103